docs/index.md

```diff
@@ -33,7 +33,7 @@ You are currently looking at the **Tutorials and Guides** for Sequelize. You mig
 const Sequelize = require('sequelize');
```

Comment:
Can we change this to `const { Sequelize, Model }`, just demonstrates the exports a little bit better.

Reply:
Good point, I think that form of importing is nice as well. Since in part this is a matter of style, and I also think that it would be good to make it clear that `Model` can be found directly in `Sequelize` (by `Sequelize.Model`), I decided to leave "both methods" in the example, one of them commented. What do you think?

I think this way the example conveys a lot of "information per line of code example", so to speak :)

Reply:
Keep in mind that `module.exports = Sequelize` and `Sequelize.Sequelize === Sequelize`, explaining that to consumers is kinda tricky.
Note: I seriously wish this wasn't done, it makes imports/exports with Sequelize a PITA to model in TS.
I think we should try to restrict documentation to only use one way to import from sequelize for consistency, but it's fair to mention that it's 'aliased'.

Reply:
I see, `Sequelize.Sequelize` is a bit odd indeed but personally I can accept it, I guess. I could also accept it not being done though. You said it's *kinda tricky* but maybe it's not that complicated. I think that if we find a quick way to pass that information, it's fine. Consumers deserve to know (unless it would get too confusing). Perhaps this doesn't have to be in the first page of the docs, though. I will make some more changes soon.

Reply:
@SimonSchick - On a second thought, would you agree with leaving this problem of "how to import sequelize" for another PR? I agree with you that the way it was done isn't so good, and that at least the documentation should take a stance on a *standard* way to do things, and this is important, but I don't want this to block the merge of my PR, since it does some other things as well.

So, I'd like to ask, would you agree with using my original version for now? At least it stays consistent with other tutorials. On the other PR, which I could help with later, we shall update the whole documentation on this matter. What do you say? I will revert my previous commit in advance. Thanks a lot.

Reply:
Yea, this would require cleanup on a bigger scale anyway, just changing it here would be more of a more of a drop in the pond.